db/migrate/20180915191418_add_super_admin_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddSuperAdminToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :super_admin, :boolean, default: false
```

Comment:
maybe we should remove the poorly named organization_admin column :)

Reply:
no keep it -- I actually reference it in a few spots.

Org Admins can do things like "invite user to their org" and "edit the org information" and any other org-specific administrative tasks.